WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.m

```diff
@@ -1104,23 +1104,32 @@ - (void)unapproveCommentWithBlock:(NotificationBlock *)block
 
 - (void)spamCommentWithBlock:(NotificationBlock *)block
 {
+    NSParameterAssert(block);
+    NSParameterAssert(self.onDeletionRequestCallback);
+    
+    // Spam Action
+    NotificationDetailsDeletionActionBlock spamAction = ^(NotificationDetailsDeletionCompletionBlock onCompletion) {
+        NSParameterAssert(onCompletion);
+        
+        NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+        CommentService *service         = [[CommentService alloc] initWithManagedObjectContext:context];
+        
+        [service spamCommentWithID:block.metaCommentID siteID:block.metaSiteID success:^{
+            onCompletion(true);
```

Comment:
Welcome back to ObjC. :)  By convention, `YES` instead of `true` and `NO` instead of `false`